src/Classes/ServiceAPI/MyRadio_Selector.php

```diff
@@ -196,11 +196,12 @@ public static function setStudio($studio)
             throw new MyRadioException('Selector Locked');
         } elseif ($response === 'ACK') {
             return [
+            'ready' => self::remoteStreams()['ready'],
```

Comment:
Well, I was going to just set it true, but that's what the other bit used. Slightly better.